invofi/apps/frontend/src/lib/migrations/002_securitization.sql

```
@@ -0,0 +1,177 @@
+  on fractionalization_records for select using (true);
+
+create policy "Originator can create fractionalization"
+  on fractionalization_records for insert
+  with check (originator_id = auth.uid());
+
+create policy "Originator can update own fractionalization"
+  on fractionalization_records for update
+  using (originator_id = auth.uid());
+
+-- Fractional positions: lender owns their row; originators can read all for their invoice
+create policy "Lender can read own positions"
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)